src/mptool.f90

```diff
@@ -18,6 +18,26 @@ module mptool
   interface pmax
     module procedure pmax_int
     module procedure pmax_mytype
+    module procedure pmax_mytype_array
+  end interface
+  interface pmin
+    module procedure pmin_mytype
+    module procedure pmin_mytype_array
+  end interface
+  !
+  interface ptabupd
+    module procedure ptable_update_int_arr
+    module procedure updatable_int
```

Comment:
Please use similar names, the current naming convention is not looking consistent compared with the other interfaces.

Reply:
done.